egg/core/interaction.py

```diff
@@ -40,14 +91,51 @@ def filtered_interaction(
             aux=aux,
         )
 
-    @classmethod
-    def minimal(cls):
-        args = [False] * 5 + [True]
-        return cls(*args)
 
-    @classmethod
-    def maximal(cls):
-        return cls()
+class RandomLogging(LoggingStrategy):
+    """
+    Log strategy based on random probability
+    """
+    def __init__(self, log_prob=1, random_seed=42, *args):
```

Comment:
Do we need a separate random seed initializer here? Wouldn't the global one suffice?

Reply:
Where would I get the global one from?

Reply:
I mean that there is already a piece of code that setups a random seed for the process, when EGG initialises. There is no need to set it up again, at least this code doesn't seem to benefit from it.

Reply:
removed in bf54f5e